internal/pkg/service/buffer/worker/service/slice_retry_test.go

```diff
@@ -104,6 +104,13 @@ func TestRetryFailedUploadsTask(t *testing.T) {
 	_, err := service.New(workerScp)
 	assert.NoError(t, err)
 
+	// Mock events send request
+	workerMock.MockedHTTPTransport().RegisterResponder(
+		http.MethodPost,
+		"https://mocked.transport.http/v2/storage/events",
+		httpmock.NewJsonResponderOrPanic(http.StatusOK, &keboola.Event{ID: "123"}),
+	)
+
```

Comment:
Ked sa nepodari poslat event je to warning ale lepsie ked tam ziaden warning v logoch nie je. Chybala tato mocked response.